variants/deploy_with_ackama_ec2_capistrano/template.rb

```diff
@@ -0,0 +1,238 @@
+puts "=" * 80
+puts "Starting variant: deploy_with_ackama_ec2_capistrano"
+puts "=" * 80
+
+copy_file "variants/deploy_with_ackama_ec2_capistrano/ackama_aws.yml", "ackama_aws.yml"
+
+append_to_file("Gemfile") do
+  <<~'EO_RUBY'
+
+    # Deployment
+    #
+    # We tell bundler not to require these gems by default because the code that
+    # requires them will explicitly require them.
+    #
+    gem "capistrano", require: false
+    gem "capistrano-bundler", require: false
+    gem "capistrano-rails", require: false
+    gem "capistrano-rbenv", require: false
+    gem "capistrano-rake", require: false
+    gem "ed25519", require: false
+    gem "bcrypt_pbkdf", require: false
+    gem 'aws_ec2_environment', git: 'https://github.com/ackama/aws_ec2_environment.git'
+
+    # capistrano-locally is required to install the app on to a freshly
+    # provisioned EC2 instance
+    gem "capistrano-locally", require: false
+
+  EO_RUBY
+end
+
+run "bundle install"
+run "bundle exec cap install"
+
+old_generated_cap_config_snippet = <<~EO_RUBY
+  set :application, "my_app_name"
+  set :repo_url, "git@example.com:me/my_repo.git"
+EO_RUBY
+
+new_ackama_cap_config_snippet = <<~EO_RUBY
+  require "dotenv/load"
+  require "aws_ec2_environment"
+
+  ##
+  # Uncomment and configure this block if your deployments go through a
```

Comment:
We shouldn't need this cause it's handled by the new `aws_ec2_environment` config

Reply:
I kept it because it just seemed like such a useful snippet to have (not entirely rationale :shrug:). This is a very focused variant so I'll remove it.